bld/namelist_files/namelist_defaults_ctsm.xml

```diff
@@ -1435,72 +1435,72 @@ lnd/clm2/surfdata_map/release-clm5.0.30/surfdata_ne0np4.CONUS.ne30x8_hist_78pfts
 <fsnowaging  >lnd/clm2/snicardata/snicar_drdt_bst_fit_60_c070416.nc</fsnowaging>
 
 <!-- Nitrogen deposition streams namelist defaults -->
```

Comment:
@wwieder , it sounds like you are talking about backwards compatibility in non-fates modes? I'm thinking maybe it would be better to change tback he following declarations to continue using the "use_cn" condition, but expand it to also include declarations with the "use_fates" conditional as well.